geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-parquet/src/main/scala/org/locationtech/geomesa/fs/storage/parquet/io/SimpleFeatureParquetSchema.scala

```diff
@@ -42,9 +43,26 @@ case class SimpleFeatureParquetSchema(sft: SimpleFeatureType, schema: MessageTyp
   lazy val metadata: java.util.Map[String, String] = Map(
     StorageConfiguration.SftNameKey -> sft.getTypeName,
     StorageConfiguration.SftSpecKey -> SimpleFeatureTypes.encodeType(sft, includeUserData = true),
-    SchemaVersionKey                -> CurrentSchemaVersion.toString // note: this may not be entirely accurate, but we don't write older versions
+    SchemaVersionKey                -> version.toString,
+    GeoParquetSchemaKey             -> null
   ).asJava
 
+  def addBoundingBoxMetadata(bbox: Envelope): java.util.Map[String, String] = {
+    val bboxString = s"[${bbox.getMinX}, ${bbox.getMaxX}, ${bbox.getMinY}, ${bbox.getMaxY}]"
+    val result = SimpleFeatureParquetSchema.geoParquetMetadata(sft) + s""","bbox":${bboxString}}}}""""
```

Comment:
we should pass the bbox into `geoParquetMetadata` function and calculate the full json there - that way the logic is all in one place, and the function is more useful b/c it returns a complete json object. we could make bbox argument an `Option[Envelope]` if you think there's a use case for creating the metadata without including the bbox.